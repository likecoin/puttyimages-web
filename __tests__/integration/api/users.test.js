const api = require('./api');

test('get users list', async () => {
  const response = await api.get('users');
  expect(response.data).toMatchSnapshot();
});

test('get user info by id', async () => {
  const response = await api.get('users/likecoin_id_1');
  expect(response.data).toMatchSnapshot();
});
