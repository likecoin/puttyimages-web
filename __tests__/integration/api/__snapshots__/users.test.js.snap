// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`get user info by id 1`] = `
Object {
  "avatar": null,
  "displayName": "Alexandre",
  "likecoinId": "likecoin_id_1",
  "wallet": "address 1",
}
`;

exports[`get users list 1`] = `
Array [
  Object {
    "avatar": null,
    "displayName": "Alexandre",
    "likecoinId": "likecoin_id_1",
    "wallet": "address 1",
  },
  Object {
    "avatar": null,
    "displayName": "Pooya",
    "likecoinId": "likecoin_id_2",
    "wallet": "address 2",
  },
  Object {
    "avatar": null,
    "displayName": "SÃ©bastien",
    "likecoinId": "likecoin_id_3",
    "wallet": "address 3",
  },
]
`;

exports[`get user info by wallet 1`] = `
Object {
  "challenge": "local",
  "wallet": "0x0000000000000000000000000000000000000000",
}
`;

exports[`post user login 1`] = `
Object {
  "avatar": null,
  "displayName": "local",
  "likecoinId": "local",
  "wallet": "0x0000000000000000000000000000000000000000",
}
`;
